metricbeat/module/aerospike/aerospike_test.go

```diff
@@ -18,9 +18,12 @@
 package aerospike
 
 import (
+	"crypto/tls"
 	"errors"
+	"fmt"
 	"testing"
 
+	"github.com/elastic/elastic-agent-libs/transport/tlscommon"
```

Comment:
Not sure what does it mea, any guidance is much appreciated

Reply:
The imports starting with `github.com/elastic/` need to appear before other 3rd party imports, run `goimports -w -local github.com/elastic metricbeat/module/aerospike`

Reply:
Thank you. Let's see if it's fixed now :)